nemo/collections/asr/parts/numba/rnnt_loss/utils/cuda_utils/gpu_rnnt_kernel.py

```diff
@@ -35,6 +35,8 @@
 
 GPU_RNNT_THREAD_SIZE = 256
 
+INF = 99999.9
```

Comment:
Number is too high for fp16 (which Numba will soon support). Can you use 10000?

Reply:
++

Reply:
Ah just saw this... by default the diff for this file is folded by GitHub... Sure, changed it.